assets/cubyz/biomes/desert/base.zig.zon

```diff
@@ -27,7 +27,6 @@
 			.id = "cubyz:simple_tree",
 			.leaves = "cubyz:air",
 			.log = "cubyz:cactus",
-			.top = "cubyz:cactus_flower",
```

Comment:
This breaks the behavior. Now cacti will no longer spawn with a flower on top.
Also should use log/cactus.

Reply:
I think you should make a new structure for this, or add an optional argument to simple structure, abusing the tree was kind of hacky to begin with.

Reply:
They should use SBBs

Reply:
I didn't see a fix for this.